web/src/components/common/DynamicTable.tsx

```diff
@@ -0,0 +1,43 @@
+import { EdsProvider, Table } from '@equinor/eds-core-react'
+
+export type TDynamicTableInput = {
+  // Keys will be used as headers. The array for each key will be the data in the column
+  [key: string]: Array<string>
+}
+export const DynamicTable = (props: {
+  input: TDynamicTableInput
+}): JSX.Element => {
+  const { input } = props
+
+  function createTableBody() {
+    const rows: Array<JSX.Element> = []
+
+    input[Object.keys(input)[0]].forEach((value, index) => {
```

Comment:
`value` is unused

Reply:
yes. But needed to access the ìndex` value